rio_tiler/mosaic/reader.py

```diff
@@ -119,8 +74,10 @@ def reader(asset: str, *args, **kwargs) -> Tuple[numpy.ndarray, numpy.ndarray]:
     assets_used: List[str] = []
 
     for chunks in _chunks(assets, chunk_size):
-        tasks = _create_tasks(reader, chunks, threads, *args, **kwargs)
-        for (t, m), asset in _filter_tasks(tasks):
+        tasks = create_tasks(reader, chunks, threads, *args, **kwargs)
+        for (t, m), asset in filter_tasks(
+            tasks, allowed_exceptions=(TileOutsideBounds,)
```

Comment:
ref #224, we allow `TileOutsideBounds` errors to be passed, they will still be logged